api/server/sdk/volume_ops.go

```diff
@@ -163,10 +165,10 @@ func (s *VolumeServer) Create(
 	}
 	source := &api.Source{}
 
-	// Update given spec according to enforcement policy set
+	// Validate/Update given spec according to default storage policy set
 	// In case policy is not set, should fall back to default way
 	// of creating volume
-	spec, err := GetEnforcedVolSpecs(locator, req.GetSpec())
+	spec, err := GetDefaultVolSpecs(ctx, req.GetSpec(), false)
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Unable to update vol specs according to policy %v", err)
```

Comment:
Make GetDefaultVolSpecs() return gRPC error style. No need to reformat here. We need GetDefaultVolSpecs() to return gRPC error style because it is the one checking for ownership and it calls stp.Inspect() which also returns in gRPC format.